core/server/master/src/main/java/alluxio/master/meta/AlluxioMasterRestServiceHandler.java

```diff
@@ -689,7 +688,6 @@ public Response getWebUILogs(@DefaultValue("") @QueryParam("path") String reques
       @DefaultValue("") @QueryParam("end") String requestEnd,
       @DefaultValue("20") @QueryParam("limit") String requestLimit) {
     return RestUtils.call(() -> {
-      FilenameFilter filenameFilter = (dir, name) -> name.toLowerCase().endsWith(".log");
```

Comment:
I know some customers/users do put potentially sensitive files under logs so we can't show them all. Could you filter by `*.log* or *.out or *.txt or *.json`?

Reply:
Done